copy_content.sh

```diff
@@ -15,3 +15,6 @@ cp -r --no-clobber imgoptim/optimized_images/. $WEBSITE/content/images
 # Copy non-optimized images (non-overwrite)
 # This makes sure that content is always available, even if it is not processed by the optimizer.
 cp -r --no-clobber content/images/. $WEBSITE/content/images
+
```

Comment:
Line 7 here (parse_content) gives content errors on the generated md files:

```
Copying content for website-pathology (group: pathology)
Error parsing content\pages\highlights\retina-arvo-2019.md.
'charmap' codec can't decode byte 0x9d in position 903: character maps to <undefined>
Error parsing content\pages\projects\ami-retina.md.
'charmap' codec can't decode byte 0x9d in position 2635: character maps to <undefined>
Error parsing content\pages\projects\brain-mri.md.
'charmap' codec can't decode byte 0x9d in position 132: character maps to <undefined>
Error parsing content\pages\projects\proacting.md.
'charmap' codec can't decode byte 0x9d in position 2152: character maps to <undefined>
Error parsing content\pages\projects\textmining.md.
'charmap' codec can't decode byte 0x9d in position 1502: character maps to <undefined>
Error parsing content\pages\publications\baid18a.md.
'charmap' codec can't decode byte 0x81 in position 1021: character maps to <undefined>
Error parsing content\pages\publications\geij18.md.
'charmap' codec can't decode byte 0x81 in position 530: character maps to <undefined>
Error parsing content\pages\publications\gonz19a.md.
'charmap' codec can't decode byte 0x9d in position 33: character maps to <undefined>
Error parsing content\pages\publications\haas18.md.
'charmap' codec can't decode byte 0x9d in position 1022: character maps to <undefined>
Error parsing content\pages\publications\herm19.md.
'charmap' codec can't decode byte 0x9d in position 1757: character maps to <undefined>
Error parsing content\pages\publications\kobu15c.md.
'charmap' codec can't decode byte 0x9d in position 2356: character maps to <undefined>
Error parsing content\pages\publications\kox11.md.
'charmap' codec can't decode byte 0x81 in position 1056: character maps to <undefined>
Error parsing content\pages\publications\litj10b.md.
'charmap' codec can't decode byte 0x81 in position 451: character maps to <undefined>
Error parsing content\pages\publications\madu13b.md.
'charmap' codec can't decode byte 0x9d in position 780: character maps to <undefined>
Error parsing content\pages\publications\mann14.md.
'charmap' codec can't decode byte 0x9d in position 977: character maps to <undefined>
Error parsing content\pages\publications\nage13.md.
'charmap' codec can't decode byte 0x9d in position 1180: character maps to <undefined>
Error parsing content\pages\publications\phil19.md.
'charmap' codec can't decode byte 0x9d in position 690: character maps to <undefined>
Error parsing content\pages\publications\post13a.md.
'charmap' codec can't decode byte 0x81 in position 917: character maps to <undefined>
Error parsing content\pages\publications\riel15.md.
'charmap' codec can't decode byte 0x9d in position 2139: character maps to <undefined>
Error parsing content\pages\publications\sard14.md.
'charmap' codec can't decode byte 0x8f in position 1562: character maps to <undefined>
Error parsing content\pages\publications\sard16.md.
'charmap' codec can't decode byte 0x81 in position 668: character maps to <undefined>
Error parsing content\pages\publications\sard17.md.
'charmap' codec can't decode byte 0x81 in position 666: character maps to <undefined>
Error parsing content\pages\publications\schm13.md.
'charmap' codec can't decode byte 0x9d in position 2192: character maps to <undefined>
Error parsing content\pages\publications\schr18c.md.
'charmap' codec can't decode byte 0x9d in position 1093: character maps to <undefined>
Error parsing content\pages\publications\silv18.md.
'charmap' codec can't decode byte 0x81 in position 2034: character maps to <undefined>
Error parsing content\pages\publications\smit19.md.
'charmap' codec can't decode byte 0x9d in position 1194: character maps to <undefined>
Error parsing content\pages\publications\stoi17a.md.
'charmap' codec can't decode byte 0x90 in position 483: character maps to <undefined>
Error parsing content\pages\publications\teuw16.md.
'charmap' codec can't decode byte 0x9d in position 636: character maps to <undefined>
Error parsing content\pages\publications\wink19.md.
'charmap' codec can't decode byte 0x81 in position 1405: character maps to <undefined>
Error parsing content\pages\publications\zaid14.md.
'charmap' codec can't decode byte 0x9d in position 2237: character maps to <undefined>
Copied pages to website-pathology.
```

Could by windows-specific.

Reply:
currently not addressed, because windows specific